File: builder-pattern/js/src/Manual.js
export class Manual {
  constructor() {
    this.name = 'PhoneName'
    this.__name__ = 'Manual'
    this.screen = [0, 0]
    this.gpuType = 0
  }

  setName(name) {
    this.name = name
  }

  getName() {
    return this.name
  }

  setScreen(size) {
    this.screen = size
  }

  getScreen() {
    return this.screen
  }

  setGpuType(type) {
    this.gpuType = type
  }

  getGpuType() {
    return this.gpuType
  }
}

File: builder-pattern/python/src/Manual.py
# -*- coding: utf-8 -*-
"""
@author: jarry
"""

class Manual():
  name = 'ManualName'
  __name__ = 'Manual'
  screen = [0, 0]
  gpuType = 0

  def __init__(self):
    pass

  def setName(self, name):
    self.name = name

  def getName(self):
    return self.name

  def setScreen(self, screen):
    self.screen = screen

  def getScreen(self):
    return self.screen

  def setGpuType(self, type):
    self.gpuType = type

  def getGpuType(self):
    return self.gpuType